libs/core/src/components/sage-avatar/sage-avatar.scss

```diff
@@ -0,0 +1,74 @@
+:host {
+  /**
+    * @prop --size: Custom size of avatar. Setting this will *not* override a set `size` prop value.
+  */
+
+  --size: var(--sage-spacing-2xl);
```

Comment:
Can you add the `display: inline-block`? When inspecting the element doesn't fill the entire parent

Reply:
Will change this.